environment.rb

```diff
@@ -0,0 +1,101 @@
+module Environment
+  class Depot
+    attr_accessor :packs
+
+    def initialize
+      @packs = {
+        simple_transportation_pack: {
+          intelligence: [:cellphone],
+          item: [
+            :medipack,
+            :chevy_versa
+          ]
+        },
+        standard_transportation_pack: {
+          intelligence: [
+            :cellphone,
+            :antenna
+          ],
+          arsenal: [
+            :colt_1911
+          ],
+          items: [
+            :handcuffs,
+            :medipack,
+            :chemistry,
+            :chevy_versa
+          ]
+        },
+        simple_mission_pack: {
+          intelligence: [
+            :infopack,
+            :laptop,
+            :cellphone,
+            :antenna
+          ],
+          arsenal: [:colt_1911, :handcuffs, :medipack, :chemistry, :financial]
+        },
+        standard_mission_pack: {
+          intelligence: [:infopack, :laptop, :cellphone, :antenna],
+          arsenal: [:remington_870, :colt_1911, :machete, :hatchet],
+          items: [:handcuffs, :medipack, :chemistry, :financial]
+        }
+      }
+    end
+  end
+
+  class Control
+    attr_accessor :missions
+
```

Comment:
Some extra empty lines may be confusing if add them or not. If you have the time, take a look of the [Rubocop](https://docs.rubocop.org/rubocop/1.23/index.html) gem, it would be great if you can install it to check the 'grammar' of your code.